apps/website/src/pages/admin/show-and-tell/review/[entryId]/preview.tsx

```diff
@@ -39,6 +39,11 @@ const AdminPreviewShowAndTellPage: NextPage<
 
   const entry = getEntry.data;
 
+  const getPostsFromANewLocation =
+    trpc.showAndTell.getPostsFromANewLocation.useQuery();
+
+  const postsFromANewLocation = getPostsFromANewLocation.data;
```

Comment:
```suggestion
  const { data: postsFromANewLocation } =
    trpc.showAndTell.getPostsFromANewLocation.useQuery();
```
Two consts seems excessive, can we not just destructure?